src/Cli/dotnet/commands/dotnet-new/LocalizableStrings.resx

```diff
@@ -308,4 +308,20 @@
     <value>Run 'dotnet workload search' to search workloads available to be installed on your SDK.</value>
     <comment>{Locked="dotnet workload search"}</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="PostAction_ModifyJson_Error_ArgumentNotConfigured" xml:space="preserve">
+    <value>{0} not configured.</value>
```

Comment:
```suggestion
    <value>Post action argument '{0}' is mandatory, but not configured.</value>
```

Reply:
Removed this localizable string and moved the (suggested) value to the Localizablestrings.resx file in `Microsoft.TemplateEngine.Cli`